sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -227,9 +242,70 @@ public EventParticipantDto saveEventParticipant(EventParticipantDto dto, boolean
 		EventParticipant entity = fromDto(dto, checkChangeDate);
 		eventParticipantService.ensurePersisted(entity);
 
+		if (existingParticipant == null) {
+			// The Event Participant is newly created, let's check if the related person is related to other events
+			// In that case, let's notify corresponding responsible Users of this relation
+			notifyEventResponsibleUsersOfCommonEventParticipant(entity, event);
```

Comment:
will take the opportunity here to mention that it is rather questionable the way sending notifications based on the feature type has been designed.. and should be addressed at some point
right now when creating a new event on a system where EVENT_PARTICIPANT_RELATED_TO_OTHER_EVENTS_NOTIFICATIONS feature is not enabled there is still a query, then some code that for each result of the query does some preparation, and then decides not to do anything with it